eHealthAfrica/aether
# Aet 165 kong internal consumer

#### What does this PR do
Configures a consumer on kong with `key-auth` plugin
Includes kong consumer and apikey as env variables on relevant containers 

#### How to test
`docker-compose -f docker-compose.yml up --build`
Check that all requests through kong `kong.aether.local/{module}/v1/` require an api key
To pass through kong, decorate your request headers with `{apikey: [KONG_APIKEY]}`
KONG_APIKEY lives in `.env` and propagated to almost all container's env variables

#### Related JIRA ticket
[AET-165](https://jira.ehealthafrica.org/browse/AET-165)
